android/app/src/main/java/com/mattermost/rnbeta/NotificationDismissService.java

```diff
@@ -4,20 +4,23 @@
 import android.content.Intent;
 import android.app.IntentService;
 import android.os.Bundle;
+import android.util.Log;
 
 import com.wix.reactnativenotifications.core.NotificationIntentAdapter;
 
 public class NotificationDismissService extends IntentService {
-
+    private Context mContext;
     public NotificationDismissService() {
         super("notificationDismissService");
     }
 
     @Override
     protected void onHandleIntent(Intent intent) {
+        mContext = getApplicationContext();
         Bundle bundle = NotificationIntentAdapter.extractPendingNotificationDataFromIntent(intent);
         int notificationId = intent.getIntExtra(CustomPushNotification.NOTIFICATION_ID, -1);
         String channelId = bundle.getString("channel_id");
-        CustomPushNotification.clearNotification(notificationId, channelId);
+        CustomPushNotification.clearNotification(mContext, notificationId, channelId);
```

Comment:
You should actually be able to pass `this` as the context instead of using the application context. Since IntentService extends Context ([link](https://developer.android.com/reference/android/app/IntentService.html))

Reply:
yes I know, but I do prefer to use the getApplicationContext() I'm not that confident doing it any other way